Appraisals

```diff
@@ -57,10 +57,9 @@ def self.gem_cucumber(version)
     #
     # DEV: Ideally, the profiler would not be loaded when running cucumber tests as it is unrelated.
     if Gem::Version.new(version) >= Gem::Version.new('4.0.0') &&
-        Gem::Version.new(version) < Gem::Version.new('7.0.0') &&
-        RUBY_PLATFORM != 'java' &&
-        Bundler::VERSION > '2.0.0'
-      gem 'google-protobuf', force_ruby_platform: true
+      Gem::Version.new(version) < Gem::Version.new('7.0.0')
```

Comment:
Indentation has taken me aback here, I thought this line was returning in void context!